pennylane/templates/layers/basic_entangler.py

```diff
@@ -120,6 +120,24 @@ def circuit(weights):
 
         Accidentally using a gate that expects more parameters throws a
         ``ValueError: Wrong number of parameters``.
+        The basic entangler can now take more than one gate as parameter for repetition. The code works as follows: For a given list of operations, rotations = [RX,RY,RZ], the model takes in parameters in shape (num_layers,num_wires*num_rotations)
+        where num_rotations = len(rotations).
+
+         .. code-block:: python
+            >>> import pennylane as qml
+            >>> from pennylane import numpy as np
+
+            >>> n_wires = 4
+            ... dev = qml.device('default.qubit', wires=n_wires)
+            ... rotations = [qml.RX, qml.RZ,qml.RY]
+            ... n_layers = 2
+            ... @qml.qnode(dev)
+            ... def circuit(weights):
+            ...     qml.BasicEntanglerLayers(weights=weights, wires=range(n_wires), rotation = rotations)
+            ...     return [qml.expval(qml.PauliZ(wires=i)) for i in range(n_wires)]
+            ... params = np.random.random(size=(n_layers, n_wires*len(rotations)))
+            >>> circuit(params)
```

Comment:
Instead of printing the circuit output, can you draw the circuit? (see [qml.draw](https://pennylane.readthedocs.io/en/stable/code/api/pennylane.draw.html))